doto-frontend/src/components/pages/Calendar/TaskScheduler.js

```diff
@@ -23,17 +23,35 @@ const addTaskToSchedule = (newTask, existingTasks, currDate) => {
 
     // Separate existing tasks with a startDate > current datetime from older tasks (which won't be considered)
     existingTasks.forEach(task => {
-        task.startDate > currDate ? competingTasks.push(task) : oldTasks.push(task);
```

Comment:
`currDate` seems to be an unused variable and can be removed (along with its JSDoc)

Reply:
Done